src/lib/ptree/ptree.lua

```diff
@@ -391,8 +371,15 @@ function Manager:sample_active_counters()
    end
 end
 
+function Manager:compute_scheduling_for_worker()
+   local ret = {}
+   for k, v in pairs(self.worker_default_scheduling) do ret[k] = v end
+   ret.cpu = self.cpuset:acquire_for_pci_addresses(self.pci_addresses)
```

Comment:
This isn't right.  One manager can manage many PCI devices on many workers, potentially on multiple NUMA nodes.  Additionally the set of PCI devices managed by a manager isn't fixed; it changes when the configuration changes.

Reply:
OK, so if the set of PCI devices can vary during runtime, it cannot be precomputed and initialized in `new_manager`. Computing the list of pci devices at runtime, as it's happening now, seems right. At this point I dunno what to do. How do you think #958 should be fixed? (Or perhaps is not worth fixing at all).